### DISCUSSION TITLE
fn + Arrows !

### DISCUSSION THREAD

**Discussion by Ahmed-Abdelgawad** [rxhanson/RectanglePro-Community#390] [General]
Hello!

I just recently moved to this app from Swish and I am really missing the option to press and hold fn + arrow keys to move windows into corners. 

Is there is a way to do this or any workarounds?

**Comment by rxhanson** [comment]
Thanks for using Rectangle Pro, and for the feedback!

The fn key is handled a little differently in macOS, and it's sort of a convention to not use it in the form of keyboard shortcuts, which is why this hasn't been added in Rectangle Pro.

I understand the desire here though, and I'll put this on my backlog (unfortunately I cannot provide a timeframe on when I can get to it though).

**Reply by LagerstedtMarcus** [reply]
I would like to up vote this aswell, used hyper dock and then fn + cmd + arrowkeys to move my windows hence that shortcut is never used. I think the recorded shortcuts should update to allow any key.
